medusa/server/api/v2/cache.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+"""Request handler for shows."""
+from tornado.escape import json_decode
+from .base import BaseRequestHandler
+from .... import db
+
+
+class SceneExceptionHandler(BaseRequestHandler):
+    """Shows request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET, OPTIONS, PUT, POST')
+
+    def get(self, *args, **kwargs):
+        """Query scene_exception information.
+
+        :param show_indexer_id: The indexer id. Like 1 for tmdb and 3 for tvmaze.
+        :param show_id:
+        :type show_indexer_id: str
+        :param season:
+        """
+        try:
+            exception_id = int(kwargs.pop('row_id', 'throwme'))
```

Comment:
You could add `row_id=None` to `get(self, ...)` and use
```
exception_id = self._parse(row_id)
```

The super class has this _parse (if I'm not mistaken) that can parse arguments and by default it uses `int`